src/hotspot/share/compiler/compilationMemoryStatistic.hpp

```diff
@@ -36,30 +36,54 @@ class outputStream;
 class Symbol;
 class DirectiveSet;
 
+// Helper class to wrap the array of arena tags for easier processing
+class ArenaTagsCounter {
```

Comment:
Sorry for being a stickler for precise names, but I would like plural for counters here - it is not a single counter, its a series/vector/array of counters.
Any of these work for me: ArenaCountersByTag - ArenaCountersByTagVector - ArenaTagCounterVector - ArenaTagCounters

Reply:
I am pretty bad in naming things, so I welcome these suggestions. I will go with ArenaCountersByTag.